LiteCore/Database/SequenceTracker.cc

```diff
@@ -217,7 +217,8 @@ namespace litecore {
                     external = i->external;
                 else if (i->external != external)
                     break;
-                changes[n++] = {i->docID, i->revID, i->sequence, i->bodySize};
```

Comment:
I'd rather avoid the extra copying. Does `changes[n++] = Change{i->docID, i->revID, i->sequence, i->bodySize}` work?

Reply:
It does not.  This seems like a compiler error, unless something in C++14 prohibits this.  The issue comes from the mismatch of types between i->docID (alloc_slice) and docID on change (slice).  A copy constructor exists between alloc_slice and slice but it seems it is not getting hit with the initializer list.  For example, this works:

```c++
slice docID = i->docID;
slice revID = i->revID;
changes[n++] = { docID, revID, i->sequence, i->bodySize };
```